js/web/lib/onnxjs/util.ts

```diff
@@ -1249,3 +1249,11 @@ export class PoolConvUtil {
 
 export const MIN_CLIP = -3.4028234663852886e+38;
 export const MAX_CLIP = 3.4028234663852886e+38;
+
+export function decodeUtf8String(buffer: Uint8Array): string {
+  if (typeof Buffer !== 'undefined' && Buffer.from) {
+    return Buffer.from(buffer.buffer, buffer.byteOffset, buffer.byteLength).toString();
```

Comment:
Is there something that is available in both Node.js and browsers ? Do we need an if...else here to handle the platforms that `Buffer` might be missing ?

Reply:
For test (ort.dev.js), we have nodejs polyfill for web. but we prefer the TextDecoder from standard WebAPI (https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder)

Reply:
Will update to use TextDecoder in both Nodejs and Web since it is universal availale